src/Microsoft.Framework.Runtime/ProjectResolver.cs

```diff
@@ -49,7 +49,7 @@ public bool TryResolveProject(string name, out Project project)
 
         private void Initialize(string projectPath, string rootPath)
         {
-            _searchPaths.Add(Path.GetDirectoryName(projectPath));
+            _searchPaths.Add(new DirectoryInfo(projectPath).Parent.FullName);
```

Comment:
`Path.GetFullPath()` doesn't help here.

To make it clear, `projectPath` is a directory, that may have a trailing slash. We want to get the parent dir of `projectPath`.

The current approach is the most readable one. If you think newing an object here may increase program startup time, I can do a naïve trailing slash removal here.